core_protocol_working_group/meetings/2024-02-22.md

```diff
@@ -1,25 +1,100 @@
-# Meeting 1
+# Inaugural Meeting [00] of the Core Protocol Working Group
 
-## Date: Feb 22nd, 2024, Thursday, 10-11 AM PST
+**Date**: February 22, 2024 [Thursday], 10-11am PST
```

Comment:
Strictly speaking we're on PT right now, but will be back on PST in May

Reply:
Ah, the time zone stuff is soooo confusing. 
This [website](https://www.timeanddate.com/time/zone/canada/vancouver) lists `PST` [Pacific Standard Time] as the current time zone and `PDT` [Pacific Daylight Time] starting from Sun, Mar 10 at 2:00 am ... hope I am reading that correctly:

> The term Pacific Time (PT) is often used to denote the local time in areas observing either [Pacific Standard Time (PST)](https://www.timeanddate.com/time/zones/pst) or [Pacific Daylight Time (PDT)](https://www.timeanddate.com/time/zones/pdt).
>
> In other words, in locations observing [Daylight Saving Time (DST)](https://www.timeanddate.com/time/dst/) during part of the year, Pacific Time is not static but switches between PST and PDT.

Reply:
My bad, thanks for setting it straight